### DISCUSSION TITLE
Alpaca-Pro 训练细节

### DISCUSSION THREAD

**Discussion by bofenghuang** [ymcui/Chinese-LLaMA-Alpaca#770] [General]
根据文档 Alpaca-Pro 与 Alpaca-Plus 一样都在 4.3M 指令数据进行精调，请问有什么具体训练细节上的差异可以增强其回复长度和质量呢？再次感谢贡献本项目！

**Comment by ymcui** [comment]
在训练策略上有所改动，后续会考虑更新在技术报告中。

**Reply by bofenghuang** [reply]
谢谢鸣哥，期待新的技术报告！
